airflow/current/Makefile

```diff
@@ -1,11 +1,11 @@
-AIRFLOW_VERSION?=2.4.1
-PYTHON_VERSION?=3.7
-COMMIT=$(shell git log -1 --pretty=%h)
-VERSION=v$(AIRFLOW_VERSION)-$(COMMIT)
-DOCKER_REPO=pubnative/airflow-2
-DOCKER_REPO_DOCKERFILE_FOLDER?=.
-DOCKER_REPO_VERSION=$(DOCKER_REPO):$(VERSION)
-DOCKER_REPO_LATEST=$(DOCKER_REPO):latest
+AIRFLOW_VERSION 				:= 2.4.1
```

Comment:
Can you rollback those ?

Reply:
any specific reason for using ?= instead of :=

Reply:
also $() is conventionally reserved for executing commands on fly and ${} is used for string substitution. Other than that nothing is changed. My IDE goes crazy and makes everything red when its out of convention

Reply:
The indentation went crazy

Reply:
`any specific reason for using ?= instead of :=`

nope, any specific reason not to?

Reply:
?= only set the value if it is not already set. which is the not the case with us, `?=` might give the wrong idea that there is another place where you can set these values

Reply:
https://www.gnu.org/software/make/manual/html_node/Setting.html#Setting 
Section 6.5

Reply:
ah this is specific to Makefile, it does not exist on bash. I am good with it. Can you fix the indentation?

Reply:
fixed indentation